scss/components/_layout.scss

```diff
@@ -5,14 +5,21 @@ body {
 .container {
     max-width: $width-container;
     margin: 0 auto;
+    padding: 0 15px;
+    min-width: 320px;
+
+    @include screen-medium {
+        box-sizing: border-box;
```

Comment:
Maybe it would be better to use `box-sizing: border-box;` by default? It related to the whole styles.

Reply:
It's more flexible, i think. If i add common box-sizing it should change styles for all elements